tests/test_mina.py

```diff
@@ -28,25 +21,16 @@ def get_nano_address_instructions(num_screen_skip):
 def get_nano_preauth_instructions(num_screen_skip):
     return get_nano_review_instructions(num_screen_skip)
 
-def get_stax_address_instructions():
-    instructions = [NavIns(NavInsID.USE_CASE_CHOICE_CONFIRM)]
-    instructions.append(NavIns(NavInsID.TOUCH, (200, 335)))
+def get_stax_address_instructions(firmware):
```

Comment:
Would probably be clearer if all devices were handled here, rather than having an `if` statement in the caller to split nano configurations and another one here to distinguish Flex and Stax :thinking:

Reply:
this part will have to be fully updated when the scenario manager will be fixed (for address review), probably in an other future PR